app/pathogen/arbovirus/dashboard/filters.tsx

```diff
@@ -136,29 +154,120 @@ export default function Filters({ excludedFields = [] }: FiltersProps) {
       ),
   });
 
+  const resetFilters = () => {
+    // Dispatch action to reset filters
+    state.dispatch({
+      type: ArboActionType.RESET_FILTERS,
+      payload: {
+        data: data ? data.records : [],
+      }, // Include an empty object as payload
+    });
+  };
+
   if (filters.isSuccess && !filters.isLoading && !filters.isError) {
     console.debug(filters.data, Object.keys(filters.data));
 
     return (
       <div>
-        <FilterSection
-          headerText="Demographic"
-          headerTooltipText="Filter on demographic variables, including population group, sex, and age group."
-          fields={demographicFilters}
-          state={state}
-          filters={filters}
-          data={data}
-          filterableFieldToLabelMap={filterableFieldToLabelMap}
-        />
-        <FilterSection
-          headerText="Study Information"
-          headerTooltipText="Filter on different types of study based metadata"
-          fields={studyInformationFilters}
-          state={state}
-          filters={filters}
-          data={data}
-          filterableFieldToLabelMap={filterableFieldToLabelMap}
-        />
```

Comment:
Heyo, I'm down for anything tbh but I would consider what I've done with `<FilterSection>` vs what's here. The 15 lines deleted here are equivalent to the 100 lines below so I would consider switching back to `<FilterSection>`

Reply:
Does this affect anything on the serotracker dashboard? after I made the changes it seemed to cause some error loading the serotracker dash. Not sure could have been something else.

Reply:
Have added it back sorry about that did not analyze it properly the first time. I agree this makes it so much cleaner thanks.